torch/_inductor/ir.py

```diff
@@ -3447,6 +3447,71 @@ def apply_constraint(self):
         self.freeze_layout_with_stride_order(self.layout.preferred_stride_order)
 
 
+class ConvolutionBinaryInplace(ExternKernelAlloc):
```

Comment:
Can we share code with `ConvolutionBinary`?

Reply:
in-place op is different from outplace cases, I think it is ok to Individually define it.